bedrock/cms/models/base.py

```diff
@@ -12,6 +12,8 @@
 
 from lib import l10n_utils
 
+from ..utils import get_locales_for_cms_page
```

Comment:
[nit, not blocking] I could be wrong, but generally it seems like we use absolute imports in bedrock for the most part? I think they are generally preferred b/c it's easier for tools and grepping etc. I'm not sure if it is still an issue today but there was also an issue with module duplication and memory when mixing relative and absolute imports.